Refactor PSP1155 to be compatible with PSP35 (#119)
Supercolony-net/openbrush-contracts: examples/psp35_extensions/batch/README.md

## PSP35 contract (ERC1155 analogue)

> Implementation of [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155) token standard with batch extension in Polkadot blockchain that allows to transfer a batch of tokens.

Reviewer: ```suggestion
Implementation of [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155) token standard with batch extension in Polkadot blockchain that allows batch transferring of tokens.
```
